build.sbt

```
@@ -1,5 +1,5 @@
 ThisBuild / organization := "io.circe"
-ThisBuild / crossScalaVersions := Seq("2.12.15", "2.13.6")
+ThisBuild / crossScalaVersions := Seq("2.12.15", "2.13.6", "3.4.0")
```

we use scala 3.3 LTS
```suggestion
ThisBuild / crossScalaVersions := Seq("2.12.15", "2.13.6", "3.3.3")
```